paddle/fluid/framework/op_registry.h

```diff
@@ -53,8 +54,9 @@ class Registrar {
 template <typename... ARGS>
 struct OperatorRegistrar : public Registrar {
   explicit OperatorRegistrar(const char* op_type) {
-    PADDLE_ENFORCE(!OpInfoMap::Instance().Has(op_type),
-                   "'%s' is registered more than once.", op_type);
+    if (OpInfoMap::Instance().Has(op_type)) {
+      PADDLE_THROW("'%s' is registered more than once.", op_type);
+    }
```

Comment:
The changes here are a bit disgusting, but I have no other way to comply with the "No PADDLE_ENROCE" rule.

Reply:
according the new rule:
```
// 判断是否true
PADDLE_ENFORCE(ctx->HasInput("X")); // 修改前
PADDLE_ENFORCE_EQ(ctx->HasInput("X"), true); //修改后
```
here can be 
```
PADDLE_ENFORCE_EQ(OpInfoMap::Instance().Has(op_type), false, "'%s' is registered more than once.", op_type);
```
although I think this isn't very good.